src/qt/forms/optionsdialog.ui

```diff
@@ -399,7 +399,7 @@
           <string>Connect to the Bitcoin network through a separate SOCKS5 proxy for Tor hidden services.</string>
          </property>
          <property name="text">
-          <string>Use separate SOCKS5 proxy to reach peers via Tor hidden services:</string>
+          <string>Use separate SOCKS&amp;5 proxy to reach peers via Tor hidden services:</string>
```

Comment:
This is being auto-assigned `S`. If we change it, `T` seems like a more logical choice...